volume/modules/local/quota.go

```diff
@@ -0,0 +1,90 @@
+package local
+
+import (
+	"sync"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// QuotaMinID represents the minimize quota id.
+	QuotaMinID = uint32(16777216)
+)
+
+var (
+	lock        sync.Mutex
+	quotaLastID uint32
```

Comment:
Could you please add more description or comments on these global variables in pkg local?
I think this is very useful for others to understand your thoughts. Actually currently I can read your code with logic, but have no idea about your thoughts.  😢